codice/ddf
# DDF-2520: Versioning Performance improvements

#### What does this PR do?

Updates versioning to prevent overhead on ingest.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?

@djblue @kcwire @jlcsmith @pklinef @coyotesqrl 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@kcwire @stustison 
#### How should this be tested?

Ingest file, modify file, overwrite content, delete, make sure history item is created for each mutable change that overwrites data (eg ingest will have no history metacard, but every update there after should create one. 
#### What are the relevant tickets?

https://codice.atlassian.net/browse/DDF-2520

!dependsOn=https://github.com/codice/ddf/pull/1367
#### Checklist:
